spec/rubocop/cop/rspec/instance_variable_spec.rb

```diff
@@ -152,5 +152,19 @@ def serialize
         end
       RUBY
     end
+
+    it 'flags an instance variable when it is also assigned ' \
```

Comment:
This is a change in behavior, and I'm not sure if it's intended one when `AssignmentOnly` is used. Anyway, that's an edge case, and I guess it's fine to flag this usage. It's worth mentioning it in the changelog though

Reply:
As I wrote in the PR description this test just documents already existing behavior, very strange as for me. I added it to ensure nothing is changed.

Reply:
Sorry, I missed that. But now I see how the existing code lead to this behaviour